yarn.lock

```diff
@@ -3462,6 +3462,25 @@
     prop-types "^15.6.1"
     react-lifecycles-compat "^3.0.4"
 
+"@react-spectrum/layout@3.0.0-alpha.1":
```

Comment:
yarn lock change is wrong, see https://github.com/adobe-private/react-spectrum-v3/pull/640/files#diff-3492513f94746f0f4e50e368715790e4R33 for fix
I don't care if we do it in this PR or in mine, might make more sense in this one

Reply:
all i did was run `yarn`; are we manually editing yarn's output?

Reply:
oh i see you upgraded

Reply:
so im happy if you upgrade first and ill rebase. let me know

Reply:
yeah, figured we shouldn't have any yarn.lock entries for our own packages ;)

Reply:
why is test utils depending on layout??

Reply:
o, it used to have a helper function for testing components with slots, however, all tests using it were removed, so we can probably delete that